.flake8.cfg

```diff
@@ -1,8 +1,8 @@
 [flake8]
-ignore = E501, E231
+ignore = E501, E231, E203, W503
```

Comment:
```
This behaviour may raise E203 whitespace before ':' warnings in style guide enforcement tools like Flake8. Since E203 is not PEP 8 compliant, you should tell Flake8 to ignore these warnings.
```

Reply:
(^ black doc)